keanu-project/src/test/java/io/improbable/keanu/KeanuRandomTest.java

```diff
@@ -54,4 +55,10 @@ public void randomNumberGenerationIsPlatformIndependent() {
         assertEquals(keanuRandom.nextDouble(), new MersenneTwister(1L).nextDouble());
         assertEquals(keanuRandom.nextDouble(), 0.41782887182714457, 1e-16);
     }
+
+    @Test
+    public void canGetStandardDeviationOfGaussian() {
+        DoubleTensor A = random.nextGaussian(new long[]{1000}).times(5);
```

Comment:
I'm not convinced that the `times(5)` adds anything to this test. Doesn't any distribution scale linearly? i.e. what you are testing here is that this particular distribution (gaussian) has a S.D. of 1.0.